lib/tessel/wifi.js

```diff
@@ -38,21 +38,20 @@ Tessel.prototype.findAvailableNetworks = function() {
 };
 
 Tessel.prototype.getWifiInfo = function() {
-  var self = this;
-  return self.simpleExec(commands.getWifiInfo())
-    .then(function fetchedWiFiInfo(resultsJSON) {
+  return this.simpleExec(commands.getWifiInfo())
+    .then((resultsJSON) => {
```

Comment:
Not important to change here, but when there is one parameter defined for an arrow function, you may omit the parens, so this can be written as: 

```js
.then(resultsJSON => {
...
}
```